src/video/cocoa/cocoa_v.mm

```diff
@@ -196,6 +196,7 @@
 
 	if ([ this->window respondsToSelector:@selector(toggleFullScreen:) ]) {
 		[ this->window performSelector:@selector(toggleFullScreen:) withObject:this->window ];
+		[NSMenu setMenuBarVisible:!full_screen];
```

Comment:
Totally contrary to any guides by Apple, we tend to have lots of spaces on Obj-C.

Also, how nice from Apple that a function with menu bar in it's name hides the dock (while the official presentation mode apparently does not).

Reply:
Sorry, my Objective C experience is, frankly, minimal!

Something like this?

```
[ NSMenu setMenuBarVisible: !full_screen ];
```

Reply:
`[ NSMenu setMenuBarVisible:!full_screen ];` i.e. spaces before and after brackets.